### DISCUSSION TITLE
Report Link only for Adminstration?

### DISCUSSION THREAD

**Discussion by teamplayer24** [liedekef/events-made-easy#484] [Q&A] [answered]
Hi Franky,

I use the function [eme_events template_id=21 template_id_header=19 template_id_footer=20]
for a list of events.

In template 21 I use the [eme_bookings_report_link]:

<tr>
<td style="border-right: 1px solid #cd0d4">#_STARTDATE</td>
<td>#_FIELD{Event_Weekday}</td>
<td>#_EVENTNAME</td>
<td>[eme_bookings_report_link template_id_header=18 template_id=17 event_id=#_EVENTID title='get']</td>
</tr>

When I'm logged in as an administrator in WP everything works as expected. I can retrieve CSV files for each event.

But as soon as I work with a normal WP user in the frontend, the link to create the CSV disappears:

View with administrator rights:
![Administrator_View](https://github.com/liedekef/events-made-easy/assets/162886228/f7edc870-5add-471d-a4b8-e7f4c856a693)

View as a normal WP user:
![Abo_WP_View](https://github.com/liedekef/events-made-easy/assets/162886228/ee6b661f-68f6-4a89-8cbf-486c60f398b6)

What am I doing wrong?

**Comment by liedekef** [comment]
The CSV is only for logged in users with the right to list bookings. If you want everyone to be able to download the csv, you need the extra option 'public_access=1'. See https://www.e-dynamics.be/wordpress/eme_bookings_report_link/

**Reply by teamplayer24** [reply]
Thanks for the really quick answer

**Reply by teamplayer24** [reply]
It works fine, thanks for your help.

One additional question:
Is it posible to store the csv-file in "utf8 with BOM"-Format (usable for Excel). I've look for some way, but i don't found a solution.

**Reply by liedekef** [reply, marked as answer]
This should help:
https://github.com/liedekef/events-made-easy/commit/372d4fdbfef65fe3d81927a9947824a7dd099b84
